### PR TITLE
osc-collab: also diff meson.build and meson_options.txt

### PR DESCRIPTION
**Pull request by DimStar77**
NOTE: only meson.build and meson_options.txt in the ROOT are diffed

### CODE REVIEW

**Comment by DimStar77** [comment]
Since more and more projects migrate to meson, diffing meson.build and meson_options.txt are becoming crucuial, so that we can track version dependency changes easily again

File: osc-collab.py [hunk 1]
```diff
@@ -2700,6 +2717,15 @@ def _collab_update(apiurl, username, email, projects, package, ignore_reserved =
...
 
+        if meson_created:
+            meson_basename = os.path.basename(meson)
+            if meson_is_diff:
+                print 'Diff in meson.build between %s and %s is available in %s' % (old_tarball, upstream_tarball_basename, meson_basename)
+            else:
+                print 'Complete meson.build of %s is available in %s' % (upstream_tarball_basename, meson_basename)
+        else:
+            print 'No meson.build information found (tarball is probably not using meson).'
+
```
Comment (vuntz) [inline osc-collab.py]: Any reason to not use `mesonopt_created`?

Comment (DimStar77) [reply osc-collab.py]: oh - indeed; mesonopt_created is missing in the output of what collab did (it's only information to the user not being printed)

Comment (DimStar77) [reply osc-collab.py]: corrected

**Pull request merged**